pkg/scheduler/cache/cache.go

```diff
@@ -51,7 +51,7 @@ func New(config *rest.Config, schedulerName string, nsAsQueue bool) Cache {
 type SchedulerCache struct {
 	sync.Mutex
 
-	kubeclient *kubernetes.Clientset
+	Kubeclient *kubernetes.Clientset
```

Comment:
I have to use clientset and recorder and I'd like to reuse libs from cache object. let me know if this is fine.